### DISCUSSION TITLE
How do you even do this

### DISCUSSION THREAD

**Discussion by smashwilson** [crudmucosa/musicrecommendations#135] [General]
This is _commitment_ ✨ 

![commitment](https://user-images.githubusercontent.com/17565/101545195-e40d5280-3974-11eb-86bd-85ae00d23525.png)

**Comment by crudmucosa** [comment]
Awww thanks. It's mostly just me wanting to share interesting music with whoever may wanna listen. Plus, it's a way for me to remember what the heck I listen to. I'm about to post today's recommendations as a matter of fact.

**Comment by smashwilson** [comment]
<img width="241" alt="image" src="https://user-images.githubusercontent.com/17565/173581463-a81bb5d3-0bb1-4233-a44a-98f32e8ab78e.png">
